src/pybind/mgr/dashboard/static/AdminLTE-2.3.7/plugins/datatables/datatables_custom_darkUI.css

```diff
@@ -1,357 +0,0 @@
-/*
```

Comment:
squash this change into 65bd5188c00a8dd68823fb5db2d85c5a573e75aa. in other words, please avoid fix-the-fix changes in a single PR.